### PR TITLE
In CSV loader options, use datetime to specify datetime type

### LINKED ISSUE #41: In the CSV loader, the datatype `date` should be `datetime`

### PR DESCRIPTION
**Pull request by xuhdev**
For consistency with pandas.

Fix #41

### CODE REVIEW

**Comment by xuhdev** [comment]
Also blocked by #37

**Review by edwardleardi — COMMENTED** [review]
LGTM, will approve once the tests pass

**Review by edwardleardi — APPROVED** [review]

**Pull request merged**
